components/formats-bsd/src/loci/formats/Memoizer.java

```diff
@@ -578,9 +576,10 @@ public boolean versionMismatch() throws IOException {
       }
 
       // REVISION NUMBER
-      if (!versionChecking && !FormatTools.VCS_REVISION.equals(revision)) {
-        LOGGER.info("Different Git version: {} not {}",
-          revision, FormatTools.VCS_REVISION);
+      if (!versionChecking &&
+          FormatTools.VERSION.endsWith("-SNAPSHOT")) {
```

Comment:
If it is something which needs checking in multiple places that would make sense.  Does any other code need to know this?